spec/nest_thermostat_spec.rb

```diff
@@ -1,9 +1,13 @@
 require 'spec_helper'
 
 module NestThermostat
-  describe Nest do
+  # Run all the requests through VCR, but don't match on the host or URI as they vary
+  # match on all the other parts of the request to find the unique request in the cassette
+  describe Nest, :vcr => { :match_requests_on => [:method, :path, :query, :body]} do
```

Comment:
Line is too long. [84/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside } missing.